app/serializers/student_comment_index_serializer.rb

```diff
@@ -0,0 +1,8 @@
+class StudentCommentIndexSerializer < BaseSerializer
+
+    attributes :id, :content, :updated_at
+
+    belongs_to :student
```

Comment:
Should be 'has_one' - this is definitely kind of confusing. Essentially, this serializer stuff only has 'has_one' and 'has_many' relationships.